manifests/operator.yaml

```diff
@@ -235,6 +273,18 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: cockroachdb
+            - name: RELATED_IMAGE_COCKROACH_v19_2_5
```

Comment:
@keith-mcclellan please double check

Reply:
the default here should be the latest releases, which are currently v20.2.5, v20.1.12, and v19.2.12  - thanks!